operations/boot_orchestration/Components.md

```diff
@@ -39,7 +39,14 @@ Stores information on the eventual desired boot artifacts and configuration for
 
 ### `enabled`
 
-If the node is enabled (enabled == True), BOS will take action to make the actual state match the desired state. Disabled nodes may receive status updates from booted nodes, but BOS will not issue power commands to the nodes while they are disabled.
+If the node is enabled (enabled == True), BOS will take action to make the actual state match the desired state. If a node is disabled (enabled == False), BOS will take no action against a node. This is an internal state
+that BOS uses for tracking whether it has finished working on a node. Typically, users should **never** enable a node (enabled == True). BOS handles this during session creation. Because BOS session cancellation is poor, users
+are forced to disable nodes (enabled == False) when they want to cancel a BOS session. Thus, while it is still uncommon, it is more likely that users will disable nodes than enable them.
```

Comment:
```suggestion
that BOS uses for tracking whether it has finished working on a node. Typically, users should **never** enable a node (enabled == True). BOS handles this during session creation.
Even if the BOS session is deleted, the nodes remain enabled in BOS, and BOS will continue to take action to make the node actual states match their desired states.
Because of this, if an administrator wishes to stop BOS from taking such actions on a node, then they must disable it (enabled == False).
Thus, while it is still uncommon, it is more likely that users will disable nodes than enable them.
```